torch/testing/_internal/opinfo/definitions/nested.py

```diff
@@ -1217,6 +1217,31 @@ def sample_inputs_nn_functional_linear(op_info, device, dtype, requires_grad, **
         )
 
 
+def sample_inputs_nn_functional_prelu(op_info, device, dtype, requires_grad, **kwargs):
+    for njt in _sample_njts(
+        device=device, dtype=dtype, requires_grad=requires_grad, dims=[3, 4]
+    ):
+        # Second dim is interpreted as number of channels; this should be non-ragged for now
+        num_channels = njt.size(1)
+        if is_nested_int(num_channels):
+            continue
+
+        weight = torch.randn(
+            num_channels,
+            device=device,
+            dtype=dtype,
+            requires_grad=requires_grad,
+        )
```

Comment:
Should we also test the case where weight can be a scalar?
https://pytorch.org/docs/stable/generated/torch.nn.functional.prelu.html

Reply:
yes! good call